### PR TITLE
test: cover PMTiles and FlatGeobuf branches

### PR DESCRIPTION
**Pull request by ibgreen**
Goals

Raise meaningful package coverage toward the 80% tranche target while keeping the required test path fast and hermetic.

Changes

- Add PMTiles metadata coverage for all supported tile MIME mappings and unknown/MLT fallbacks.
- Cover PMTiles TileJSON extraction, optional format-header retention, and invalid metadata handling.
- Add FlatGeobuf malformed-header coverage for truncation, invalid magic, and unsupported versions.
- Cover the no-geometry fast path without requiring a fixture.

Validation

- yarn lint fix
- yarn test-headless modules/pmtiles/test/pmtiles-metadata.spec.ts modules/flatgeobuf/test/reader-coverage.spec.ts
- 31 tests passed in 1.94s
- yarn test-audit: 609 files, 0 Tape, 45 skipped, 0 violations
- git diff --check

The tranche intentionally uses compact unit and boundary cases rather than adding large fixtures or slow integration matrices.

### CODE REVIEW

File: modules/csv/src/papaparse/papa-parser.ts [hunk 1]
```diff
@@ -132,7 +135,7 @@ export class ChunkStreamer {
       this._completed = true;
     }
 
-    // if (!finishedIncludingPreview && (!results || !results.meta.paused)) this._nextChunk();
+    if (!finishedIncludingPreview && (!results || !results.meta.paused)) this._nextChunk();
```
[the remark this answers is not in the document]
Comment (ibgreen) [reply modules/csv/src/papaparse/papa-parser.ts:138]: Addressed in ac8908be4c. StringStreamer now drains fixed-size string chunks iteratively behind a re-entrancy guard, eliminating recursive stack growth. Added a regression parsing 12,000 rows with chunkSize: 1.

File: modules/csv/src/papaparse/papa-parser.ts [hunk 2]
```diff
@@ -283,13 +289,13 @@ export class ParserHandle {
     this._paused = true;
     this._parser.abort();
     this._input = this._input.substr(this._parser.getCharIndex());
+    this.streamer._partialLine = '';
```
[the remark this answers is not in the document]
Comment (ibgreen) [reply modules/csv/src/papaparse/papa-parser.ts]: Addressed in ac8908be4c. Pause now retains the partial line only when it contains an unmatched quote, preserving quoted rows across resume while avoiding duplication for ordinary rows. Added a regression for a quoted row split at a chunk boundary and reran the existing PapaParse suite.
